### ISSUE TITLE
XAML Preview is not showing anything

### ISSUE THREAD

**Issue by LWChris** [Live-Charts/Live-Charts#1004]
#### How to reproduce?

I have copied the [Basic Column example](https://lvcharts.net/App/examples/v1/wpf/Basic%20Column), but in the XAML preview, none of the chart elements are rendered. I just see blank space. When the app is executed, the chart displays fine including all additional styling I applied, but it's really hard to style the graph and axes etc. when you don't have any preview to see what it does.

#### Extra notes

* I expected to see some kind of dummy series
* Clicking on the area in the preview selects the `CartesianChart` element in the XAML, so the chart is actually there
* I can also set the `ChartesianChart.Background` to see the whole area the control consumes; since that works I know the chart streches as intended
* The `d:DesignInstance` is defined and set to the view model with the properties
* The issue persists through a full VS restart

#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>v0.9.7</td>
</tr>
<tr>
    <td>.NET Framework Version (System)</td>
    <td>4.8.03752</td>
</tr>
<tr>
    <td>.NET Framework Version (App Target)</td>
    <td>4.6.2</td>
</tr>
<tr>
    <td>Windows</td>
    <td>Windows 10 Pro Version 1903 (Build 18362.418)</td>
</tr>
<tr>
    <td>VS Version</td>
    <td>Visual Studio Professional 2017 Version 15.9.16</td>
</tr>
</table>

**Comment by Mrxx99** [comment]
Have you set 'IsDesignTimeCreatable' to true for the DesignInstance?

**Comment by LWChris** [comment]
Yes, it is set to "True", which is also correct because the ctor doesn't take any arguments.
